test/e2e/pages/vl-cookie-consent.page.js

```diff
@@ -7,64 +7,72 @@ class VlCookieConsentPage extends Page {
         await super.load(Config.baseUrl + '/demo/vl-cookie-consent.html');
     }
 
-    async _getConsent(selector) {
-        const element = await this.driver.findElement(By.css(selector));
+    async _getConsent(number) {
+        const element = await this.driver.findElement(By.css(`#cookie-consent-${number}`));
```

Comment:
Ik ben niet enorm fan van deze manier, maar goed, mag gerust zo blijven.

Reply:
Van de manier waarop het was? Ik heb hier alleen de duplicatie weg gewerkt. Waar ben je wel fan van?

Reply:
id's een volwaardige naam geven

Reply:
Aangezien de page objecten hier wel een zinvolle naam hebben, vond ik het op zich niet zo erg. Het aanpassen is niet veel werk, dus zal ik voorzien.